assembl/static2/js/app/components/debate/multiColumns/tabbedColumns.jsx

```diff
@@ -28,8 +29,8 @@ export default class TabbedColumns extends React.Component {
     const col = messageColumns[index];
     const synthesisProps = showSynthesis && {
       classifier: activeKey,
-      synthesisTitle: I18n.t('multiColumns.synthesis.title', { colName: col.name }),
-      synthesisBody: col.header || I18n.t('multiColumns.synthesis.noSynthesisYet'),
+      synthesisTitle: get(col, 'columnSynthesis.subject') || I18n.t('multiColumns.synthesis.title', { colName: col.name }),
```

Comment:
why don't you use the third param of `lodash.get` (default value) ?

Reply:
Oh right, I forgot about that.

Reply:
done in https://github.com/assembl/assembl/commit/af5de5f637e848283f0ea55af700483a5094ddb6